README.md

```diff
@@ -15,16 +14,16 @@ Carlo communicates with the locally-installed browser instance using the [Puppet
 <!-- [START usecases] -->
 ###### What can I do?
 
-With Carlo, you can create hybrid applications that use Web stack for rendering and Node for capabilities:
-- For Node applications, you can visualize dynamic state of your Node app using web rendering stack
-- For Web applications, you can expose additional system capabilities accessible from Node
-- You can bundle your application into a single executable using [pkg](https://github.com/zeit/pkg).
+Carlo creates hybrid applications that use Web stack for rendering and Node for capabilities:
```

Comment:
Carlo does not create applications, it is a library. User is creating applications with Carlo.

Reply:
Switched to "With Carlo users can create"